app/src/main/res/values/styles.xml

```diff
@@ -278,15 +288,21 @@
     <style name="SubtitleTextStyle" parent="TextAppearance.MaterialComponents.Body1">
         <item name="android:textColor">?secondaryText</item>
         <item name="android:textSize">14sp</item>
+        <item name="android:fontFamily">@font/metropolis_semibold</item>
+        <item name="fontFamily">@font/metropolis_semibold</item>
     </style>
 
     <style name="Subtitle12TextStyle" parent="TextAppearance.MaterialComponents.Body1">
         <item name="android:textColor">?secondaryText</item>
         <item name="android:textSize">12sp</item>
+        <item name="android:fontFamily">@font/metropolis_semibold</item>
+        <item name="fontFamily">@font/metropolis_semibold</item>
     </style>
 
     <style name="ToolbarTitleTextStyle" parent="HeaderTextStyle">
         <item name="android:textSize">20sp</item>
+        <item name="android:fontFamily">@font/metropolis_semibold</item>
+        <item name="fontFamily">@font/metropolis_semibold</item>
```

Comment:
@ekager apparently `fontFamily` needs a different namespace on different APIs, so we wind up double declaring it all over here.  Do you know if there's any better way to mix it into all of these styles?

Reply:
Totally missed this. @kglazko after our chat this week - is this still an issue? Does just using `fontFamily` (using the app namespace) not work on all API levels?